### ISSUE TITLE
Workpress API needs to be HTTPS

### ISSUE THREAD

**Issue by gmh04** [edina/CuriousEdinburgh2#21]
that is: http://curiousedinburgh.org/wp-json.  This is especially for IOS that has become more strict. 

I suppose this depends on who maintains the wordpress, a question for Nicola.

**Comment by NicolaOsborne** [comment]
The wordpress is currently sitting on AWS. It is managed by the project team (academic) in terms of content and tours but by EDINA in terms of set up, API, etc. There is an issue with access at present that is being resolved by the Infrastructure team.

**Comment by NicolaOsborne** [comment]
To be done when VM restarted.

**Comment by gmh04** [comment]
Problem creating QuoVadis SSL certificate for curious edinburgh domain.  Delay will mean no HTTPS URL for worpress api for this release.

**Comment by NicolaOsborne** [comment]
@gmh04 @jollopre Does this still need resolving? And how high priority is it?

**Comment by gmh04** [comment]
Yes it still needs resolving, an email regarding SSL certificate for curious edinburgh domain to Infrastructure in March was never replied to.  Not sure how high a priority this is, maybe @jollopre knows.

**Comment by NicolaOsborne** [comment]
@gmh04 @jollopre I have followed this up and Ben S will follow up and advise on progress obtaining certs.

**Comment by NicolaOsborne** [comment]
Note: some mixed content issues with stylesheet and images that are on http rather than https. This won't cause an issue with the app I think, but may cause an issue with browser warnings.
